diff --git a/dist/assistant/data/chatStream.js b/dist/assistant/data/chatStream.js
index 6eef71bbe7ee63f7ff0d2dd84eaa580b5abe5db4..69df112092b6ea6bbe0f4f38130cb8584d86ac3a 100644
--- a/dist/assistant/data/chatStream.js
+++ b/dist/assistant/data/chatStream.js
@@ -4,7 +4,6 @@ exports.chatStream = void 0;
 const assistant_data_1 = require("../../pinecone-generated-ts-fetch/assistant_data");
 const utils_1 = require("../../utils");
 const errors_1 = require("../../errors");
-const node_stream_1 = require("node:stream");
 const chat_1 = require("./chat");
 const chatStream = (assistantName, apiProvider, config) => {
     return async (options) => {
@@ -29,7 +28,7 @@ const chatStream = (assistantName, apiProvider, config) => {
             }),
         });
         if (response.ok && response.body) {
-            const nodeReadable = node_stream_1.Readable.fromWeb(response.body);
+            const nodeReadable = response.body;
             return new utils_1.ChatStream(nodeReadable);
         }
         else {
